content/integrations/azure.md

```diff
@@ -73,3 +121,4 @@ Please make sure you have enabled diagnostics on your VMs. Diagnostics allows VM
    [2]: https://portal.azure.com
    [3]: mailto:support@datadoghq.com
    [4]: https://management.azure.com
+   [5]: https://manage.windowsazure.com
```

Comment:
@irabinovitch talked to @tmichelet about it. We moved the azure metadata to each sub integration, so the overall azure integration isn't associated w/ a specific CSV file